rdtools/filtering.py

```diff
@@ -31,13 +31,14 @@ def normalized_filter(normalized, low_cutoff=0.01, high_cutoff=None):
     return (normalized > low_cutoff) & (normalized < high_cutoff)
 
 
-def poa_filter(poa, low_irradiance_cutoff=200, high_irradiance_cutoff=1200):
+def poa_filter(poa_global, low_irradiance_cutoff=200,
+               high_irradiance_cutoff=1200):
```

Comment:
Not sure if the cutoff names should be changed.  Maybe `poa_global_low`?

Reply:
I'm ok with keeping this poa.  poa_global isn't strictly required here.